### PR TITLE
Snyk Organization Support

### PR DESCRIPTION
**Pull request by PeyGis**
# Description

We have improved on the way in which users export the Snyk ocean integration. With this updates, users can fetch multiple organization's data into Port with one installation.

## Updated docs pages

docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk.md

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk.md [hunk 1]
```diff
@@ -85,7 +85,8 @@ integration:
     type: POLLING
   config:
   // highlight-next-line
-    organizationId: SNYK_ORGANIZATION_ID
+    organizationId: SNYK_ORGANIZATION_ID  ## Required when you want to fetch data for a single organization
+    groups: SNYK_GROUPS  ## Required when you want to fetch data for all organizations within the specified group(s)
```
Comment (Tankilevitch) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk.md:89]: I think the default should be without those options, and rather have in the note / guide on where to add it if they want to manage the scope which the integration will have access to

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk.md [hunk 2]
```diff
@@ -227,7 +228,8 @@ of `Secret Text` type:
 | Parameter                                     | Description                                                                                                                                                      | Required |
 | --------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
 | `OCEAN__INTEGRATION__CONFIG__TOKEN`           | The Snyk API token                                                                                                                                               | ✅       |
-| `OCEAN__INTEGRATION__CONFIG__ORGANIZATION_ID` | The Snyk organization ID                                                                                                                                         | ✅       |
+| `OCEAN__INTEGRATION__CONFIG__ORGANIZATION_ID` | The Snyk organization ID                                                                                                                                         | ❌      |
```
Comment (Tankilevitch) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk.md:231]: It is unclear why and when to use it. You want to emphasis that it should be passed if wanting to restrict the access to specific organizationId, do the same for the groups config.

**Review by Tankilevitch — CHANGES_REQUESTED** [review]
Nice job

Still some changes required.
Also the integration guide misses any information about what will be the default behavior of the installations and how to manage it, I think it require a section of its own.

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk.md [hunk 3]
```diff
@@ -285,6 +281,18 @@ pipeline {
...
+By default, the integration fetches all organizations associated with the provided Snyk token. If you wish to customize access, the following parameters are available:
+
+`OCEAN__INTEGRATION__CONFIG__ORGANIZATION_ID`: Use this parameter to restrict access to a specific organization. If specified, the integration will fetch data only for the provided organization.
+
+`OCEAN__INTEGRATION__CONFIG__GROUPS`: When you want to limit access to all organizations within specific Snyk groups, use this parameter. Provide a comma-separated list of Snyk group IDs, and the integration will filter data accordingly.
+
+:::note
+If neither parameter is provided, the integration will operate with the default behavior of fetching all organizations associated with the supplied Snyk token.
+:::
+
```
Comment (Tankilevitch) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk.md:294]: I think this should be before the installation

**Review by Tankilevitch — CHANGES_REQUESTED** [review]

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk.md [hunk 4]
```diff
@@ -5,14 +5,25 @@ import AdvancedConfig from '../../../generalTemplates/_ocean_advanced_configurat
...
```
Comment (Tankilevitch) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk.md:26]: This should be inside the installation section.

Please make sure that it aligned for each kind of the installations with the right params and where to set them.

**Review by Tankilevitch — CHANGES_REQUESTED** [review]

**Review by Tankilevitch — APPROVED** [review]

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk.md [hunk 5]
```diff
@@ -73,8 +86,33 @@ To install the integration using ArgoCD, follow these steps:
 1. Create a `values.yaml` file in `argocd/my-ocean-snyk-integration` in your git repository with the content:
 
 :::note
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk.md:88]: Please add a title for the admonition (and for all future admonitions), it makes it more inviting to actually read it and not skip it.

:::note title-here
content
:::

**Review by hadar-co — APPROVED** [review]

**Pull request merged**